src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -45,6 +48,10 @@ public class AntiforgeryOptions
         /// <para>
         /// <see cref="CookieBuilder.SameSite"/> defaults to <see cref="SameSiteMode.Strict"/>.
         /// <see cref="CookieBuilder.HttpOnly"/> defaults to <c>true</c>.
+        /// <see cref="CookieBuilder.IsEssential"/> defaults to <c>true</c>. If this is marked as <c>false</c>, then
+        /// this cookie token is not sent in the response and any further requests to the server which require
+        /// antiforgery validation would fail due to this missing cookie token.
```

Comment:
This isn't really accurate.

I'd suggest something like:
``
The cookie used but the antiforgery system is part of a security system that is necessary when using cookie-based authentication. It should be consider required for the application to function.
```